docs/integrations/sources/jira-migrations.md

```diff
@@ -1,5 +1,31 @@
 # Jira Migration Guide
 
+## Upgrading to 2.0.0
+
+Note: this change is only breaking if you are using the `Issues`, `Boards Issues` or `Sprint Issues` streams in Incremental Sync mode.
+
+This is a breaking change because the migration to low-code will result in incremental stream states changing their format by saving them per partition. Additionally, an issue in the `Project Avatars` stream has been fixed, where a field was defined as 'project_id,' when in reality, it holds a project key (now it holds project id). Please follow the instructions below to migrate to version 2.0.0:
+
```

Comment:
To gracefully handle these changes for your existing connections, we highly recommend resetting your data before resuming your data syncs with the new version.

Reply:
Done